cockpit/IPC/src/views/Configurator.tsx

```diff
@@ -74,11 +51,14 @@ export default function Configurator() {
 
   // Get data and schema for each name and store in states
   useEffect(() => {
-    console.log('names: ', names);
     if (names.length > 0) {
-      console.log('names > 0 ');
       names.forEach((name: string) => {
-        fetchDataFromDBus(name).then(({ parsedData, parsedSchema }) => {
+        fetchDataFromDBus(
+          name, // process name
+          name, // interface name
```

Comment:
Is the process name and interface name the same?

Reply:
Yes, for example the state machine process:
```com.skaginn3x.config.test_rig.def.test_rig_config```
has only one interface of the same name.

Is that what you intended? @jbbjarnason @omarhogni